ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/MigratorHandler.java

```diff
@@ -0,0 +1,314 @@
+package uk.gov.hmcts.cmc.ccd.migration.services;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.ccd.migration.ccd.services.CoreCaseDataService;
+import uk.gov.hmcts.cmc.ccd.migration.idam.models.User;
+import uk.gov.hmcts.cmc.ccd.migration.stereotypes.LogExecutionTime;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.CountyCourtJudgmentType;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponseType;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.FormaliseOption;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseAcceptation;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+import uk.gov.hmcts.cmc.domain.models.offers.StatementType;
+import uk.gov.hmcts.cmc.domain.models.response.DefenceType;
+import uk.gov.hmcts.cmc.domain.models.response.FullDefenceResponse;
+import uk.gov.hmcts.cmc.domain.models.response.ResponseType;
+import uk.gov.hmcts.cmc.domain.utils.PartyUtils;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Predicate;
+
+import static uk.gov.hmcts.cmc.domain.models.response.DefenceType.ALREADY_PAID;
+import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.NO;
+
+@Service
+public class MigratorHandler {
```

Comment:
How about renaming `MigratorHandler` to `MigrationHandler` ?

Reply:
done